ChangeLog

```diff
@@ -9,6 +9,9 @@ Bugfix
    * Fix mbedtls_x509_crt_profile_suiteb, which used to reject all certificates
      with flag MBEDTLS_X509_BADCERT_BAD_PK even when the key type was correct.
      In the context of SSL, this resulted in handshake failure. #1351
+   * MD functions deprecated in 2.7.0 are no longer inline, to provide
+     a migration path for alternative implementations without a sudden
+     ABI break.
```

Comment:
I am not fully in line with this ChangeLog entry: It only mentions alternative implementations, but the ABI break affects everybody who uses Mbed TLS as a dynamically linked library and calls the deprecated functions directly.

Reply:
@hanno-arm Ok. How about “to facilitate the transition through the ABI change”?

I don't want to imply that we normally promise ABI compatibility — we don't (except on patch releases), but we do try to be accommodating and this case was easy to paper over.

And I did wonder: should it be classified as “Bugfix” or “Changes”? Cc @Patater

Reply:
@gilles-peskine-arm I'm ok with that formulation. 

I'd classify it as a "Change" or "ABI Change"